src/test/java/sorinfoca/seleniumbasicspart2/JavaExceptions.java

```diff
@@ -0,0 +1,4 @@
+package sorinfoca.seleniumbasicspart2;
```

Comment:
You can catch an exception in each of the classes from Tema6 package. No need to add a separated class. Thanks!